lib/mirage_key.ml

```diff
@@ -343,6 +343,14 @@ module V6 = struct
 
 end
 
+let resolver ?(default = Ipaddr.V4.of_string_exn "91.239.100.100") () =
+  let doc = Fmt.strf "DNS resolver (default to anycast.censurfridns.dk)" in
```

Comment:
Could we expose this default as a value in the resolver library in the future, rather than duplicating it here?

Reply:
this sounds preferable, but I do not know how with the current mirage and functoria design -- anyone has an example somewhere how to do this?